assets/js/index.js

```diff
@@ -16,6 +16,7 @@ typeahead.loadjQueryPlugin();
 
 $('.typeahead').each(function applyTypeahead() {
   var $el = $(this);
+  $el.parent().addClass('typeahead');
   var $parent = $el.parent();
```

Comment:
```js
$parent.addClass('typeahead');
```
???

Also, is "typeahead" the best class to apply? (I'm envisaging future bugs in which jQuery works slightly differently and this causes recursion up the the root element.) Would "hidden-hint" be better?